tests/Unit/ProtoTests/BasicOneof/out/src/Gapic/BasicOneofGapicClient.php

```diff
@@ -212,16 +205,28 @@ public function __construct(array $options = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function aMethod($extraDescription, $extraSummary, $extraRequest, $extraIndex, $extraDouble, $extraFloat, $extraBool, array $optionalArgs = [])
+    public function aMethod($supplementaryData, array $optionalArgs = [])
     {
         $request = new Request();
-        $request->setExtraDescription($extraDescription);
-        $request->setExtraSummary($extraSummary);
-        $request->setExtraRequest($extraRequest);
-        $request->setExtraIndex($extraIndex);
-        $request->setExtraDouble($extraDouble);
-        $request->setExtraFloat($extraFloat);
-        $request->setExtraBool($extraBool);
+        if ($supplementaryData->isExtraDescription()) {
+            $request->setExtraDescription($supplementaryData->getExtraDescription());
+        } elseif ($supplementaryData->isExtraSummary()) {
+            $request->setExtraSummary($supplementaryData->getExtraSummary());
+        } elseif ($supplementaryData->isExtraRequest()) {
+            $request->setExtraRequest($supplementaryData->getExtraRequest());
+        } elseif ($supplementaryData->isExtraIndex()) {
+            $request->setExtraIndex($supplementaryData->getExtraIndex());
+        } elseif ($supplementaryData->isExtraDouble()) {
+            $request->setExtraDouble($supplementaryData->getExtraDouble());
+        } elseif ($supplementaryData->isExtraFloat()) {
+            $request->setExtraFloat($supplementaryData->getExtraFloat());
+        } elseif ($supplementaryData->isExtraBool()) {
+            $request->setExtraBool($supplementaryData->getExtraBool());
+        } else {
+            throw new ValidationException("A field for the oneof supplementary_data must be set in param $supplementaryData");
+        }
+
+        
```

Comment:
This extra newline can go.

Reply:
The whitespacing in generated files isn't within our control, and it would take significant refactoring to fix this. See also [this change](https://github.com/googleapis/gapic-generator-php/commit/03d04a7d2ec5b94ff4e8c82eb52d5d8c62594c8f#diff-b9c6528c2b224211ab9a88e3ac0916511fb5b122ee5f3fec118ea8ae72e4dfcd) where we attempted to fix an indentation issue (which is part of, but not the only objective, of the change).